src/components/FromSTIX/FromSTIX.jsx

```diff
@@ -6,7 +6,7 @@ import Statistics from "./Statistics";
 import { getDataForStatistics, getOfficialFieldsFromMapping } from "./utils";
 import { requiredStixFields } from "../../global/requiredStixFields";
 
-const FromSTIX = () => {
+const FromSTIX = ({isShown}) => {
```

Comment:
The prop `isShown` to the `FromSTIX` component is confusing, I mean, what is shown? moreover, to be consistent, use the `redux` way to handle state (useSelector for `state.fromStix` and `state.toStix`), that way we manage the state. (read about `react-redux`)